laravel-app/database/migrations/2024_12_05_120809_create_merch_items_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('merch_items', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('merch_id');
+            $table->unsignedInteger('lang_id');
+            $table->string('name');
+            $table->text('detail');
+            $table->timestamps();
+        });
```

Comment:
- languagesテーブルを作ることを懸念してカラム名の変更
- 外部キーはひとまずunsignedBigIntegerを採用（時間があれば変える）

```suggestion
        Schema::create('merch_items', function (Blueprint $table) {
            $table->id();
            $table->unsignedBigInteger('merch_id');
            $table->unsignedBigInteger('language_id');
            $table->string('name');
            $table->text('detail');
            $table->timestamps();
        });
```